lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -29,7 +29,7 @@ const DEFAULT_RETRIES = 0;
  * @typedef SmokehouseResult
  * @property {string} id
  * @property {boolean} success
- * @property {Array<{passed: number, failed: number, log: string}>} expectationResults
+ * @property {Array<{url: string, passed: number, failed: number, log: string}>} expectationResults
```

Comment:
```suggestion
 * @property {Array<{requestedUrl: string, passed: number, failed: number, log: string}>} expectationResults
```

to be consistent/clear what's being returned here